www/vanv08/hw08/homework.js

```diff
@@ -0,0 +1,202 @@
+// 0 ukol
+
+console.log('Chaotic good');
+console.log('Nenapadl mne appendovat text do results, tak jsem to delal pres input fields');
+
+
+
+// Prvni ukol
+
+function logPepeAge() {
+    var pepeAge = null;
+    if (!!pepeAge) {
+        console.log('Pepe\'s age is: ' + pepeAge);
+        document.getElementById("pepe_field").value = pepeAge;
+    } else {
+        console.log('I dont know Pepe\'s age)');
+        document.getElementById("pepe_field").value = 'I dont know Pepe\s age';
+    }
+}
+
+var age = document.getElementById("task-1");
+age.addEventListener("click", logPepeAge, true);
+
+// Druhy ukol jedno z nich musi byt null aby se mohla vypocitat ta druha
+
+function tempConverter() {
+    var celsius_value = document.getElementById("celsius_value").value;
+    var fahrenheiht_value = document.getElementById("fahrenheiht_value").value;
+
+    function celsiusToF(celsius_value) {
+        return ((celsius_value * 9) / 5) + 32;
+    }
+
+    function fahrenheihtToC(fahrenheiht_value) {
+        return ((fahrenheiht_value - 32) * 5) / 9;
+    }
+
+    if (!!fahrenheiht_value && !celsius_value) {
+        console.log('Fahrenheit to celsius is: ' + fahrenheihtToC(fahrenheiht_value).toFixed(2) + 'C');
+        document.getElementById("temperature").value = parseFloat(fahrenheihtToC(fahrenheiht_value).toFixed(2)) + 'C';
```

Comment:
`document.getElementById("temperature")` provest jednou a ulozit do promenne

Reply:
document.getElementId("temperature") se uz zobrazuje jen jednou, ale nejsem si jisty zda je to opraveno spravne